pimd/pim_igmp_join.h

```diff
@@ -26,7 +26,8 @@
 #define SOL_IP IPPROTO_IP
 #endif
 
-#ifndef MCAST_JOIN_SOURCE_GROUP
+#ifndef MCAST_JOIN_GROUP
```

Comment:
Please have two different #ifndef and #endif's 
one does not necessariy mean the other

Reply:
Separated #ifndefs